src/core_plugins/kibana/public/dev_tools/partials/dev_tools_app.html

```diff
@@ -1,15 +1,15 @@
 <div class="dev-tools-app-container app-container">
-  <nav class="navbar navbar-default navbar-static-top subnav">
-    <bread-crumbs omit-current-page="true"></bread-crumbs>
-
-    <ul class="nav navbar-nav">
-      <li ng-repeat="item in devTools" ng-class="{active: currentPath === item.url}">
-        <a class="navbar-link" kbn-href="{{::item.url}}">
-          {{::item.display}}
-        </a>
-      </li>
-    </ul>
-  </nav>
-
+  <bread-crumbs class="bread-crumbs--navbar" omit-current-page="true"></bread-crumbs>
+  <kbn-top-nav name="devtools">
+    <div class="localTabs">
+      <a
+        ng-repeat="item in devTools"
+        class="localTab"
+        ng-class="{'localTab-isSelected': currentPath === item.url}"
+        kbn-href="{{::item.url}}">
```

Comment:
Would you mind putting this closing bracket on the next line, at the same depth as the opening bracket ([per style guide](https://github.com/elastic/kibana/blob/master/style_guides/html_style_guide.md))?

```html
<a
  attributes
>
```